porch/api/porch/types_packagerevisions.go

```diff
@@ -128,6 +131,23 @@ type PackageCloneTaskSpec struct {
 	Strategy PackageMergeStrategy `json:"strategy,omitempty"`
 }
 
+type PackageUpdateTaskSpec struct {
+	// // `Subpackage` is a path to a directory where to clone the upstream package.
+	// Subpackage string `json:"subpackage,omitempty"`
+
+	// `Upstream` is the reference to the upstream package to clone.
+	Upstream UpstreamPackage `json:"upstreamRef,omitempty"`
+
+	// 	Defines which strategy should be used to update the package. It defaults to 'resource-merge'.
+	//  * resource-merge: Perform a structural comparison of the original /
+	//    updated resources, and merge the changes into the local package.
+	//  * fast-forward: Fail without updating if the local package was modified
+	//    since it was fetched.
+	//  * force-delete-replace: Wipe all the local changes to the package and replace
+	//    it with the remote version.
+	Strategy PackageMergeStrategy `json:"strategy,omitempty"`
```

Comment:
I'm planning to change this struct to be empty and use the Upstream and Strategy from the package revision's clone task instead.